libfastboot/flash.c

```diff
@@ -461,8 +463,9 @@ static struct label_exception {
 	{ L"ioc", flash_ioc },
 #endif
 #ifdef BOOTLOADER_POLICY
-	{ CONVERT_TO_WIDE(ACTION_AUTHORIZATION), authenticated_action }
+	{ CONVERT_TO_WIDE(ACTION_AUTHORIZATION), authenticated_action },
 #endif
+	{ L"keybox", flash_keybox }
```

Comment:
Can we define a macro to control the availability of "fastboot flash keybox" command?

if the feature is not enabled, we should not expose the interface.